pages/apply.html

```diff
@@ -211,50 +182,79 @@
-            </li>
-            <li>
-              <input type="radio" name="entry.305267568" value="5-point preference based on active duty in the U.S. Armed Forces (TP)." id="group_305267568_3" role="radio" class="ss-q-radio" aria-label="5-point preference based on active duty in the U.S. Armed Forces (TP)."  aria-required="true">
-              <label for="group_305267568_3">5-point preference based on active duty in the U.S. Armed Forces (TP).</label>
-            </li>
-            <li>
-              <input type="radio" name="entry.305267568" value="10-point preference based on a compensable service connected disability of at least 10% but less than 30% (CP)." id="group_305267568_4" role="radio" class="ss-q-radio" aria-label="10-point preference based on a compensable service connected disability of at least 10% but less than 30% (CP)." aria-required="true">
-              <label for="group_305267568_4">10-point preference based on a compensable service connected disability of at least 10% but less than 30% (CP).</label>
-            </li>
-            <li>
-              <input type="radio" name="entry.305267568" value="10-point preference based on a compensable service connected disability of 30% or more (CPS)." id="group_305267568_5" role="radio" class="ss-q-radio" aria-label="10-point preference based on a compensable service connected disability of 30% or more (CPS)." aria-required="true">
-              <label for="group_305267568_5">10-point preference based on a compensable service connected disability of 30% or more (CPS).</label>
-            </li>
-            <li>
-              <input type="radio" name="entry.305267568" value="10-point preference for non-compensable disability or Purple Heart (XP)." id="group_305267568_6" role="radio" class="ss-q-radio" aria-label="10-point preference for non-compensable disability or Purple Heart (XP)." aria-required="true">
-              <label for="group_305267568_6">10-point preference for non-compensable disability or Purple Heart (XP).</label>
-            </li>
-            <li>
-              <input type="radio" name="entry.305267568" value="10-point preference based on widow/widower or mother of a deceased veteran, or spouse or mother of a disabled veteran (XP)." id="group_305267568_7" role="radio" class="ss-q-radio" aria-label="10-point preference based on widow/widower or mother of a deceased veteran, or spouse or mother of a disabled veteran (XP)." aria-required="true">
-              <label for="group_305267568_7">10-point preference based on widow/widower or mother of a deceased veteran, or spouse or mother of a disabled veteran (XP).</label>
-            </li>
-          </ul>
+          <p>Please see &quot;Government pay grades explained:&quot; <a href="https://www.google.com/url?q=https://pages.18f.gov/joining-18f/pay-grades/&amp;sa=D&amp;usg=AFQjCNHPdTF2o1CKuVPZtq_dRjd8jNlsvw">https://pages.18f.gov/joining-18f/pay-grades/</a></p>
```

Comment:
This should be a direct link instead of a Google result. We can actually just link the words 'Government pay grades explained' now that we have access to the magic of HTML. :rocket: